notebooks/bimodal_dist_plot.ipynb

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.    linewidth = 2
</pre><p>Apply directly if not using at multiple places. If using at multiple places, define on the top as a constant (LINE_WIDTH)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/pyprobml-fork/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>